app/src/main/java/com/standuptracker/service/PersonService.kt

```diff
@@ -8,18 +8,17 @@ import com.standuptracker.dto.Person
 import retrofit2.Call
 import retrofit2.Callback
 import retrofit2.Response
-import java.lang.Exception
 
 class PersonService {
-    fun fetchPeople() : MutableLiveData<ArrayList<Person>> {
+    fun fetchPeople(): MutableLiveData<ArrayList<Person>> {
         var _people = MutableLiveData<ArrayList<Person>>()
         val service = RetrofitClientInstance.retrofitInstance?.create(IPersonDAO::class.java)
         val call = service?.getAllPeople()
         // must put call on the background thread
-        call?.enqueue(object: Callback<ArrayList<Person>> {
+        call?.enqueue(object : Callback<ArrayList<Person>> {
             override fun onFailure(call: Call<ArrayList<Person>>, t: Throwable) {
-                throw Exception("failed to read JSON")
                 Log.d("JSON", "Save Failed")
+                throw Exception("failed to read JSON")
```

Comment:
I moved this line because the `Log.d` statement was after the `throw` statement, which would make the `Log.d` statement unreachable. The `throw` ends the function call and stops any further code from executing.

Reply:
Oh yes.  Good catch here!